Source/Configuration.swift

```diff
@@ -4,53 +4,55 @@ public struct Configuration {
 
   // MARK: Colors
 
-  public static var backgroundColor = UIColor(red: 0.15, green: 0.19, blue: 0.24, alpha: 1)
-  public static var gallerySeparatorColor = UIColor.black.withAlphaComponent(0.6)
-  public static var mainColor = UIColor(red: 0.09, green: 0.11, blue: 0.13, alpha: 1)
-  public static var noImagesColor = UIColor(red: 0.86, green: 0.86, blue: 0.86, alpha: 1)
-  public static var noCameraColor = UIColor(red: 0.86, green: 0.86, blue: 0.86, alpha: 1)
-  public static var settingsColor = UIColor.white
-  public static var bottomContainerColor = UIColor(red: 0.09, green: 0.11, blue: 0.13, alpha: 1)
+  public var backgroundColor = UIColor(red: 0.15, green: 0.19, blue: 0.24, alpha: 1)
+  public var gallerySeparatorColor = UIColor.black.withAlphaComponent(0.6)
+  public var mainColor = UIColor(red: 0.09, green: 0.11, blue: 0.13, alpha: 1)
+  public var noImagesColor = UIColor(red: 0.86, green: 0.86, blue: 0.86, alpha: 1)
+  public var noCameraColor = UIColor(red: 0.86, green: 0.86, blue: 0.86, alpha: 1)
+  public var settingsColor = UIColor.white
+  public var bottomContainerColor = UIColor(red: 0.09, green: 0.11, blue: 0.13, alpha: 1)
 
   // MARK: Fonts
 
-  public static var numberLabelFont = UIFont(name: "HelveticaNeue-Bold", size: 19)!
-  public static var doneButton = UIFont(name: "HelveticaNeue-Medium", size: 19)!
-  public static var flashButton = UIFont(name: "HelveticaNeue-Medium", size: 12)!
-  public static var noImagesFont = UIFont(name: "HelveticaNeue-Medium", size: 18)!
-  public static var noCameraFont = UIFont(name: "HelveticaNeue-Medium", size: 18)!
-  public static var settingsFont = UIFont(name: "HelveticaNeue-Medium", size: 16)!
+  public var numberLabelFont = UIFont(name: "HelveticaNeue-Bold", size: 19)!
```

Comment:
@richardtop care to also use system fonts, as you described here https://github.com/hyperoslo/ImagePicker/issues/256 😉